Vagrantfile

```diff
@@ -48,22 +48,29 @@ echo "Launching bootstrap"
 exec "/srv/scality/metalk8s-$SHORT_VERSION/bootstrap.sh"
 SCRIPT
 
+NETWORK_RANGE = "192.168.42."
+BOOTSTRAP_IP_OFFSET = 10
+
 # To support VirtualBox linked clones
 Vagrant.require_version(">= 1.8")
 
 Vagrant.configure("2") do |config|
-  config.vm.define :bootstrap do |bootstrap|
-    bootstrap.vm.box = "centos/7"
-    bootstrap.vm.box_version = "1811.02"
+  config.vm.box = "centos/7"
+  config.vm.box_version = "1811.02"
+
+  config.vm.provider "virtualbox" do |v|
+    v.linked_clone = true
+    v.memory = 2048
+    v.cpus = 2
+  end
+
+  config.vm.define :bootstrap, primary: true do |bootstrap|
     bootstrap.vm.hostname = "bootstrap"
 
-    bootstrap.vm.provider "virtualbox" do |v|
-      v.linked_clone = true
-      v.memory = 2048
-      v.cpus = 2
+    bootstrap_ip = NETWORK_RANGE + "#{BOOTSTRAP_IP_OFFSET}"
+    bootstrap.vm.network "private_network", ip: bootstrap_ip
 
-      bootstrap.vm.synced_folder ".", "/vagrant", type: "virtualbox"
-    end
+    bootstrap.vm.synced_folder ".", "/vagrant", type: "virtualbox"
```

Comment:
This is VirtualBox-specific, hence needs to be in a `provider` block.

Reply:
Indeed.